recipe/meta.yaml

```
@@ -1,40 +1,45 @@
-{% set version = "3.1.43" %}
+{% set name = "GitPython" %}
+{% set version = "3.1.45" %}
 
 package:
-  name: gitpython
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/G/GitPython/GitPython-{{ version }}.tar.gz
-  sha256: 35f314a9f878467f5453cc1fee295c3e18e52f1b99f10f6cf5b1682e968a9e7c
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
+    sha256: 85b0ee964ceddf211c41b9f27a49086010a190fd8132a24e21f362a4b36a791c
+  - url: https://github.com/gitpython-developers/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 67a52b150bf206612fefa478e6e5d5a31a3a379cd346577a386194878b9f8762
+    folder: gh
 
 build:
   number: 0
-  skip: True  # [py<37]
+  skip: true  # [py<37]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python
     - pip
+    - python
     - setuptools
```

```suggestion
    - setuptools
    - wheel
```
This logic was moved out of wheel and into setuptools in v70.1.0, and any other dependency that setuptools has on wheel is done by vendoring (copying the code directly).
Use wheel, as we don`t have a lower bound for setuptools here.